ccm_web/server/migrations/2021.05.18T18.16.39.create-user.ts

```diff
@@ -0,0 +1,46 @@
+import { DataTypes } from 'sequelize'
+import { Migration } from '../migrator'
+
+export const up: Migration = async ({ context: sequelize }) => {
+  await sequelize.getQueryInterface().createTable('user', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: DataTypes.INTEGER
+    },
+    firstName: {
+      allowNull: true,
+      type: DataTypes.STRING
```

Comment:
So, I'm guessing you determined the default length of `255` would work for us here and with the other fields? That seems okay to me.

Reply:
We discussed on this in the meeting, so resolving now